### PR TITLE
App tests

### PR DESCRIPTION
**Pull request by tajemouti**
## Hi Reviewer 👋 ##
Below is a list of the accomplished tasks while working on this milestone:

1. Created a feature branch for this milestone.
2. Created unit tests for the Logic modules.
3. Implemented unit tests for components.
4. Added tests for pages' components.
5. Edited the README file.
6. Fixed linters' errors.

### CODE REVIEW

File: src/tests/Button.test.js
Comment (Efiamotu-1) [inline src/tests/Button.test.js:1]: - [ ] From the project requirement displayed in the image below you are not writing test for snapshots and userEvents in your test files, I kindly suggest you go over this recommended [youtube video](https://www.youtube.com/watch?v=2NxjGhNjmA4) from minute `13.30` to see how to add snapshots test.
mi

File: src/tests/operate.test.js
Comment (Efiamotu-1) [inline src/tests/operate.test.js:1]: - [ ] I also suggest you add more test to show more operations asides the addition which you have done here 👍

**Review by Efiamotu-1 — CHANGES_REQUESTED** [review]
Hi @tajemouti & @Graciano1997 & @firstpj , 

 

Good job so far! 👏. Your Code is Awesome! 💪

There are some **few** issues 🐛 that you still need to work 🔧 on though to go to the next project ⏭️ but you are almost there! 🏃
## Don't Worry 🙏

It's ok to have bugs in your code 🐛. Please, see this code review as a **chance** for you to **learn** practical coding skills and learn **new tips** that can improve your code quality 💯 and help you **get a better ** 💼.

_I am going to guide you on how to fix them down below. 🤓_

## Highlights :dart: 
- ✔️ No linter Errors
- ✔️ Descriptive README
- ✔️ Descriptive PR title and description
- The code is top-notch! 💯 and has a huge potential.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

> **Note:** Please, don't Merge This Pull Request yet. Only Merge when it is Approved ✅.


## Thank you for your time 🙏 and call me if you need help or clarification ☎️

- This is My **GitHub**: [@Efiamotu-1](https://github.com/Efiamotu-1/)
- This is My **Twitter**: [@EFYAMOTU](https://twitter.com/EFYAMOTU)
- This is My **linkedIn**.[Habeeb Efiamotu Musa](https://www.linkedin.com/in/musa-habeeb/)
- This is My **Slack**: [Habeeb Efiamotu Musa](https://slack.com)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/tests/Button.test.js
Comment (tajemouti) [reply src/tests/Button.test.js:1]: Hi @Efiamotu-1,
Thank you for the insightful feedback.. We really appreciate it..
Cheers

**Review by anagudelogu — APPROVED** [review]
# STATUS: APPROVED! 🍾 🎆 🥇 

Hi @tajemouti, 
I'm Andres.

You did a great job working on this milestone, your project is complete! 🚀 
There is nothing else to say other than... it's time to merge it 🤖  
Congratulations! 🎉 



## Highlights
✅ Great use of Git flow. 💯 
✅ Your project is documented professionally. 📜 
✅ Descriptive PR title, summary and commit messages. 💬 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/tests/operate.test.js
Comment (firstpj) [reply src/tests/operate.test.js:1]: I also thank you very much for the review, and the change recommended, it really helped us to learn new things.

**Comment by Graciano1997** [comment]
@anagudelogu  thanks very much for recognizing all our efforts!

**Comment by Graciano1997** [comment]
@tajemouti  very well Done! Happy code

**Pull request merged**